### PR TITLE
Work item link tabs fixed

### LINKED ISSUE #583: Header links of link-items should provide detail for respective links
Header links of link-items should provide detail for respective links

Suppose we have Headers in details view , which have 5 columns (types of links) as blocked by, tests etc...
header
When we click on "blocked by" it should display details for only blocked by link items, Right now its displaying overall items for every link.

Can you provide some inputs for this?

### LINKED ISSUE #586: "blocks" shows up on dropdown: link item
"blocks" shows up on dropdown: link item

Video:  https://drive.google.com/a/redhat.com/file/d/0B1qUq_laG6COQXBaR2ZncW5pNFE/view?usp=sharing

@sanbornsen , plz look into it.

### PR DESCRIPTION
**Pull request by sanbornsen**
- fixes #583
- fixes #586

---
This change is [Reviewable](https://reviewable.io/reviews/almighty/almighty-ui/607)

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html [hunk 1]
```diff
@@ -1,18 +1,24 @@
 <div class="link-container" id="wi-link">
-    <div class="link-header" (click)="toggleLinkComponent()">
-        <span class="content__title pull-left">
+    <div class="link-header">
+        <span class="content__title pull-left"
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html]: Any particular reason to give class name in this format("content__title").

**Comment by SMahil** [comment]
This does not fix UI issue. Below is the screenshot

screen shot 2017-01-02 at 4 03 03 pm

For 4 or more than 4 items UI is still breaking.

**Comment by sanbornsen** [comment]
I don't think this PR is to fix this issue. Can you give me the issue of you are talking about?

**Comment by SMahil** [comment]
@sanbornsen Yes for the same WI linking UI is breaking but I guess We need to break down the issue. This PR will fix the functionality. May be we can mention this in commit message.
For UI Issue I will fix the CSS

File: src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html]: I don't know.

**Pull request merged**